### PR TITLE
Disable review button when captcha is on

### PR DESCRIPTION
**Pull request by tonytan4ever**

### CODE REVIEW

**Comment by tonytan4ever** [comment]
1) . Fixed
2) . Can't reproduce this.  When you submitted it moves to next open item.  So you will see in "Edit", but it's next open item.

File: concordia/views.py [hunk 1]
```diff
@@ -238,29 +238,28 @@ class ConcordiaItemView(TemplateView):
     template_name = "transcriptions/item.html"
 
     def get_context_data(self, **kws):
+        from .serializers import ItemSerializer
 
-        response = requests.get(
```
Comment (acj3rd) [inline concordia/views.py]: Why was the REST API call removed here? All database action is supposed to be through REST api calls. If and existing REST api call does not exist, write a new one.

Comment (tonytan4ever) [reply concordia/views.py]: See the other PR:  @acdha is working on:
https://github.com/ArtemisConsulting/concordia/pull/109

Without this there will be 500 ERROR for Asset page. We talked about this earlier this week.

Comment (rstorey) [reply concordia/views.py]: We'll eventually run the REST API in a separate docker container. At that point, we'll use front end code in the UI to invoke the REST API.

Comment (acj3rd) [reply concordia/views.py]: sounds good, I'll approve then

**Pull request merged**